src/main/java/org/folio/search/utils/CallNumberUtils.java

```diff
@@ -0,0 +1,182 @@
+package org.folio.search.utils;
+
+import static java.util.Collections.unmodifiableMap;
+import static java.util.Locale.ROOT;
+import static java.util.stream.Collectors.joining;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import lombok.experimental.UtilityClass;
+import org.apache.commons.lang3.RegExUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.util.Assert;
+
+@UtilityClass
+public class CallNumberUtils {
+
+  private static final int CN_MAX_CHARS = 10;
```

Comment:
![image](https://user-images.githubusercontent.com/94473337/235745349-fb1fe3f6-766e-4c0e-8a20-55971e14d7a2.png)

Reply:
In method on line 139 there is 'CN_MAX_CHARS-1'